Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -108,7 +108,7 @@ public ActionsTreeView(VisualElement root, StateContainer stateContainer)
 
             m_ActionsTreeView.selectedIndicesChanged += indicies =>
             {
-                var index = indicies.First();
+                var index = indicies.FirstOrFallback(-1); 
```

Comment:
What about
`var index = indicies.Any() ? indicies.First() : -1;`
instead of making extension funcions?

Also, not part of this PR but `indicies` should be `indices`.

Reply:
Agree with @jamesmcgill here

Reply:
Using Any() makes a lot of sense, however it is a little bit unreliable and will generate a warning since e.g. `var index = indicies.Any() ? indicies.First() : -1;` could possibly result in multiple-enumeration which isn't something guaranteed by IEnumerable<T> as a generic interface. In this particular situation its still safe since the underlying implementation is a List enumerator, but it becomes implementation dependent if UITK at any point would replace it by an enumerator not allowing multiple-enumeration. Still want me to change? 
This is the whole reason why I added an extension not having that problem.

Reply:
I will begin updating the typo though until we just settle on this detail.

Reply:
I didn't see a warning but of course, if you are getting one then it's not good and you should go with your solution in that case.

Reply:
This is no longer relevant since FirstOrFallback was removed from this PR so marking this resolved.